TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/RobotTools/TelemetryDump.java

```diff
@@ -1,48 +1,36 @@
 package org.firstinspires.ftc.teamcode.rework.RobotTools;
 
 import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.teamcode.rework.ModuleTools.TelemetryProvider;
 
+import java.util.ArrayList;
 import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
 
 public class TelemetryDump {
```

Comment:
I think TelemetryDump shouldn't be a static object. I think it's better to have it instantiated in robot and passed to each module?